### DISCUSSION TITLE
Share page

### DISCUSSION THREAD

**Discussion by boxadesign** [ONSdigital/design-system#1190] [Feedback]
Use this place to discuss the [share page component](https://service-manual.ons.gov.uk/design-system/components/share-page) in the [ONS Design System](https://service-manual.ons.gov.uk/design-system)

**Comment by Al-Davies** [comment]
**UR findings on share**
tested as part of end-to-end Census testing Feb-Sept 2020 & CMD on ONS website - 2019

- users unlikely to share anything of a government nature to their social media accounts - ie. such as they have completed the Census
- need to be clear over exactly what is going to be shared (a link to a file, a download, their personal information etc)
- being able to share business data or information (such as CMD customisations) would be useful to circulate to colleagues
- users noted share options for Census when they were tested
